bindings/gumjs/gumquickmemory.c

```diff
@@ -834,6 +837,28 @@ gum_quick_memory_write (JSContext * ctx,
   return result;
 }
 
+GUMJS_DEFINE_FUNCTION (gum_quick_memory_read_volatile_array)
+{
+  gpointer address;
+  gssize length = -1;
+  gsize n_bytes_read = -1;
+  guint8 * buffer_data;
+
+  if (!_gum_quick_args_parse (args, "pZ", &address, &length))
+    return JS_EXCEPTION;
+
+  if (length == 0)
+    return JS_NULL;
+
+  buffer_data = gum_memory_read (address, length, &n_bytes_read);
+  
+  if (buffer_data == NULL)
+    return _gum_quick_throw_literal (ctx, "Memory read failed");
```

Comment:
```suggestion
    return _gum_quick_throw_literal (ctx, "memory read failed");
```

(JavaScript error messages should start in lowercase -- unlike GLib error messages, which start capitalized.)